CHANGELOG.md

```diff
@@ -6,8 +6,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## x.y.z
 
 ### Added
-
-### Changed
```

Comment:
Could you restore the 'Changed' section here?

We'll remove it when making a release if it is still empty, but for now it acts as a marker/placeholder.